### DISCUSSION TITLE
Rough idea of production-readiness timeline

### DISCUSSION THREAD

**Discussion by csmcle** [cdmoro/bootstrap-vue-3#555] [General]
Yes, I'm aware this is a ridiculous question at this stage of the project. But I have someone in a professional context who wants to hang their hat on waiting for this to be ready and I'd appreciate if someone could confirm for me if there's any likelihood of a production release, say, within the next year.

**Comment by VividLemon** [comment]
Actually, last night, there has been discussions among the collaborators and I about a 1.0 release. There are a few components in native bootstrap that remain. Other than those, the biggest issue is finishing the documentation and writing unit tests. One collaborator had mentioned that at least 50% coverage would be sufficient. 

In summary, the components that are done, will see little change, but will probably still have a few changes, the documentation will need to be updated to reflect the current changes, and the majority of the work remaining is creating tests. 

If your friend feels incapable of helping your context, anyone is free to help build the library.

**Reply by csmcle** [reply]
Thanks :)
